File: Outras telas/testeGrafico/app/src/main/java/com/example/ejemplographview/MainActivity.java
package com.example.ejemplographview;

import androidx.appcompat.app.AppCompatActivity;

import android.graphics.Color;
import android.os.Bundle;

import com.jjoe64.graphview.GraphView;
import com.jjoe64.graphview.ValueDependentColor;
import com.jjoe64.graphview.series.BarGraphSeries;
import com.jjoe64.graphview.series.DataPoint;

public class MainActivity extends AppCompatActivity {

    GraphView grafica;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        grafica = findViewById(R.id.grafica);

        BarGraphSeries<DataPoint> series = new BarGraphSeries<>(new DataPoint[]{
           new DataPoint(0, 1),
           new DataPoint(1, 5),
           new DataPoint(2, 3),
           new DataPoint(3, 2),
           new DataPoint(4, 6)

        });

        grafica.addSeries(series);

        series.setValueDependentColor(new ValueDependentColor<DataPoint>() {
            @Override
            public int get(DataPoint data) {
                return Color.rgb((int)data.getX()*255/4, (int) data.getY()*255/6, 100);
            }
        });

        series.setSpacing(20);

        series.setDrawValuesOnTop(true);
        series.setValuesOnTopColor(Color.BLUE);
    }
}

File: Outras telas/MultiplasTelas/app/src/main/java/bandtec/com/multiplastelas/MainActivity.kt
package bandtec.com.multiplastelas

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View


class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun irTela2 (v: View){
        val tela2 = Intent(this, Tela2::class.java)

        tela2.putExtra("Usuário", "zeruela")
        tela2.putExtra("Idade", 30)
        tela2.putExtra("isadmin", true)

        startActivity(tela2)
    }

    fun sair (v: View){
        finishAffinity()
    }
}
